turbo/snapshotsync/freezeblocks/block_reader.go

```diff
@@ -253,24 +260,36 @@ func (r *RemoteBlockReader) EventsByBlock(ctx context.Context, tx kv.Tx, hash co
 	return result, nil
 }
 
-func (r *RemoteBlockReader) LastEventID(_ kv.RwTx) (uint64, error) {
+func (r *RemoteBlockReader) LastFrozenEventId() uint64 {
 	panic("not implemented")
 }
 
-func (r *RemoteBlockReader) LastFrozenEventID() uint64 {
+func (r *RemoteBlockReader) Span(_ context.Context, _ kv.Getter, _ uint64) ([]byte, error) {
 	panic("not implemented")
 }
 
-func (r *RemoteBlockReader) Span(_ context.Context, _ kv.Getter, _ uint64) ([]byte, error) {
+func (r *RemoteBlockReader) LastSpanId(_ context.Context, _ kv.Tx) (uint64, bool, error) {
 	panic("not implemented")
 }
 
-func (r *RemoteBlockReader) LastSpanID(_ kv.RwTx) (uint64, bool, error) {
+func (r *RemoteBlockReader) LastFrozenSpanId() uint64 {
 	panic("not implemented")
 }
 
-func (r *RemoteBlockReader) LastFrozenSpanID() uint64 {
-	panic("not implemented")
+func (r *RemoteBlockReader) LastMilestoneId(ctx context.Context, tx kv.Tx) (uint64, bool, error) {
+	return 0, false, fmt.Errorf("not implemented")
+}
+
+func (r *RemoteBlockReader) Milestone(ctx context.Context, tx kv.Getter, spanId uint64) ([]byte, error) {
+	return nil, nil
+}
+
+func (r *RemoteBlockReader) LastCheckpointId(ctx context.Context, tx kv.Tx) (uint64, bool, error) {
+	return 0, false, fmt.Errorf("not implemented")
+}
+
+func (r *RemoteBlockReader) Checkpoint(ctx context.Context, tx kv.Getter, spanId uint64) ([]byte, error) {
+	return nil, nil
```

Comment:
maybe best to panic with not implemented rather than silently returning empty? unless this is used?